### DISCUSSION TITLE
Simplelogin.io limits and thresholds

### DISCUSSION THREAD

**Discussion by EthraZa** [simple-login/app#675] [Q&A] [answered]
Hi.

Looking at the docs and pricing page there is no mention I could find about any limits to protect Simplelogin from abuse, but I guess they may exist.
Other places have limits on the amount of messages that can be received per minute, hour or day.
There is such a limit at Simplelogin.io or any other rate limit that we may hit at any time?

Thanks.

**Comment by nguyenkims** [comment, marked as answer]
@EthraZa there are some abuse detections in SimpleLogin indeed. We don't want to explicitly document these limits as the detection algorithm can change and that can give info to abusers :).

**Reply by EthraZa** [reply]
Fair enough
